maintenance/ebpf/ebpf-and-eks.md

```diff
@@ -184,26 +176,141 @@ which is suitable:
     ```
-    
-  * Restart any pods that were created before {{site.prodname}} was installed.  Typically, this is only `kube-dns`:
-  
-    ```
-    kubectl delete pod -n kube-system -l k8s-app=kube-dns
-    ```
+
+* Create a nodegroup, using the AMI ID you noted above.
+  * `--node-ami` should be set to the AMI ID of the image built above.
+  * `--node-ami-family` should be set to `Ubuntu1804` (despite the upgrade).
+
+  For example:
+  ```
+  eksctl create nodegroup \
+    --cluster my-calico-cluster \
+    --node-type t3.medium \
+    --node-ami auto \
+    --max-pods-per-node 100 \
+    --node-ami-family Ubuntu1804 \
+    --node-ami <AMI ID>
+  ```
 
 %>
 {% endtabs %}
 
-* Continue with the instructions in the main [Enabling eBPF page](./enabling-bpf).  
-
-  When configuring {{site.prodname}} to connect to the API server, use the load balanced domain name created by EKS.
-  One way to determine the domain name is to extract it from `kube-proxy`'s config map:
+#### Configure {{site.prodname}} to connect directly to the API server
+* When configuring {{site.prodname}} to connect to the API server, we need to use the load balanced domain name
+  created by EKS.  It can be extracted from `kube-proxy`'s config map by running:
   ```
   kubectl get cm -n kube-system kube-proxy -o yaml | grep server
   ```
-  should show the server name, for example:
+  which should show the server name, for example:
   ```
       server: https://d881b853ae9313e00302a84f1e346a77.gr7.us-west-2.eks.amazonaws.com
   ```
-  In that case you should use `d881b853ae9313e00302a84f1e346a77.gr7.us-west-2.eks.amazonaws.com` for `KUBERNETES_SERVICE_HOST`
+  In this example, you would use `d881b853ae9313e00302a84f1e346a77.gr7.us-west-2.eks.amazonaws.com` for `KUBERNETES_SERVICE_HOST`
   and `443` (the default for HTTPS) for `KUBERNETES_SERVICE_PORT` when creating the config map.
+
+  Create the following config map in the `kube-system` namespace using the host and port determined above:
+
+  ```
+  kubectl apply -f - <<EOF
+  kind: ConfigMap
+  apiVersion: v1
+  metadata:
+    name: kubernetes-services-endpoint
+    namespace: kube-system
+  data:
+    KUBERNETES_SERVICE_HOST: "<API server host>"
+    KUBERNETES_SERVICE_PORT: "443"
+  EOF
+  ```
+
+* Wait 60s for kubelet to pick up the `ConfigMap` (see Kubernetes [issue #30189](https://github.com/kubernetes/kubernetes/issues/30189){:target="_blank"}); then, restart the {{site.prodname}} pods to pick up the change:
+
+  ```
+  kubectl delete pod -n kube-system -l k8s-app=calico-node
+  ```
+
+  And, if using Typha and/or calico-kube-controllers (if you're not sure if you're running these, run the commands
+  anyway, they will fail with "No resources found" if the pods aren't present):
+  ```
+  kubectl delete pod -n kube-system -l k8s-app=calico-typha
+  kubectl delete pod -n kube-system -l k8s-app=calico-kube-controllers
+  ```
+
+* Confirm that pods restart and then reach the `Running` state with the following command:
```

Comment:
Confirm that pods restart and reach the `Running` state with the following command: